linux_os/guide/system/software/integrity/disable_prelink/rule.yml

```diff
@@ -1,5 +1,7 @@
 documentation_complete: true
 
+prodtype: ol7,rhel6,rhel7,rhv4,sle15
```

Comment:
Not sure about `rhv4` and `sle15` products. I don't know how to verify if `rhv4` that we support has the `prelink` package but I don't see the package in SLE15, see https://packagehub.suse.com/search/?q=prelink

Reply:
`rhv4` is built off of `rhel8` so you can remove it. Pretty sure that dropping prelink from `sle15` is a safe bet. It isn't part of openSUSE either: https://software.opensuse.org/package/prelink